scripts/fp_assessments_sum.R

```diff
@@ -0,0 +1,52 @@
+# this script gets a total count of fish passage field assessments done
+
+# xref my_crossings pscis, pull records from bcdata using record ID
+
+get_this <- bcdata::bcdc_tidy_resources('7ecfafa6-5e18-48cd-8d9b-eae5b5ea2881') %>%
+  filter(bcdata_available == T)  %>%
+  pull(package_id)
+
+dat <- bcdata::bcdc_get_data(get_this)
+
+## xref_pscis_my_crossing_modelled
+xref_pscis_my_crossing_modelled <- dat %>%
+  purrr::set_names(nm = tolower(names(.))) %>%
+  dplyr::filter(consultant_name == "IRVINE") %>%
+  sf::st_drop_geometry() %>%
+  tidyr::separate(assessment_date, into = c('year', 'month', 'day'), remove = F)
+
+# total count by year for all pscis assessments and habitat confirmations (before unsubmitted 2022 sites added)
+xref_total_count <- xref_pscis_my_crossing_modelled %>%
+  group_by(year) %>%
+  summarise(total = n())
+
+# total count by year for all phase 2 habitat confirmations
+hab_con_count <- xref_pscis_my_crossing_modelled %>%
+  filter(str_detect(funding_project, 'Fish Habitat Confirmations|Phase 2')) %>%
+  group_by(year) %>%
+  summarise(total = n()) %>%
+  add_row() %>%
```

Comment:
can avoid a line of code with a case_when below with `add_row(year = '2022', total = NA_real_)`